### DISCUSSION TITLE
squashfs image vs NFS chroot

### DISCUSSION THREAD

**Discussion by Mogaba** [ltsp/ltsp#712] [Q&A] [answered]
What are pros and cons of using NFS export of chroot over squashfs image?

**Comment by alkisg** [comment, marked as answer]
\+ Squashfs image is about 5 times faster than a chroot,
\- but it requires `ltsp image` which takes a few minutes,
\- along with some disk space,
\+ on the other hand, updating an NFS chroot might cause the clients to experience some instability since the chroot is the lower part of the client overlayfs (writeable rootfs, with tmpfs being the upper part) and it's not supposed to change.

So in general, IF you are using chroots at all, it's best to use them for quick testing, while for production use `ltsp image` to publish chroots as squashfs images.
